packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -213,8 +226,34 @@
                 >
                     <oui-tile-description>
                         <span
+                            class="mb-1"
+                            data-translate=""
```

Comment:
Is it used ?

Reply:
No, removed here: https://github.com/ovh/manager/pull/8392/commits/765411a3700a87f578beeb60e6948438fab567c2